static/sass/styles.scss

```diff
@@ -829,4 +829,11 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
   @media screen and (min-width: $breakpoint-small) and (max-width: $breakpoint-large - 1) {
     padding-bottom: 0;
   }
-}
\ No newline at end of file
+}
+
+// Vertical align grid contents
+.u-equal-height-item {
```

Comment:
Looks like your not using this is the markup. Can it be removed?

Reply:
Removed.